massa-execution-worker/src/execution.rs

```diff
@@ -762,6 +761,17 @@ impl ExecutionState {
         )
     }
 
+    /// Get every datastore key of the given address.
+    pub fn get_every_final_datastore_key(&self, addr: &Address) -> Vec<Vec<u8>> {
+        self.final_state
+            .read()
+            .ledger
+            .get_entire_datastore(addr)
+            .into_iter()
+            .map(|v| v.0)
```

Comment:
I don't understand if we have all the keys and the values here hy we return only keys.

Reply:
I thought it's because it would be heavy as a return payload to have everything but maybe I'm wrong.